app/javascript/packs/application.js

```diff
@@ -0,0 +1,20 @@
+// Polyfill modern JS features, like promises, async functions, etc.
+import "core-js/stable";
+import "regenerator-runtime/runtime";
```

Comment:
Could you explain these in some more detail? I am not familiar with how these work. My assumption is that these imports are needed so that the packs that webpacker generates are compatible with older browsers, correct? How does this relate to `.browserslistrc` – I thought that the browsers list also informs webpack (?) or babel (?) what level of compatibility is needed in the code that it generates.

Reply:
From what I've read, these two imports are a direct replacement to `import "@babel/polyfill";` (if you've ever done that). They are meant to include only the necessary polyfills based on `@babel/preset-env` (a dependency of `@rails/webpacker`) - and `.browserlistrc` is the way to list the browsers you want to support. It _sounds_ like all the tools in the chain have matured to the point that this will do what you want: not import unnecessary code.